app/javascript/components/visualization/ScatterPlot.js

```diff
@@ -62,7 +62,7 @@ function RawScatterPlot({
 
       const perfTimeFrontendStart = performance.now()
 
-      Plotly.newPlot(graphElementId, scatter.data, layout)
+      Plotly.react(graphElementId, scatter.data, layout)
```

Comment:
I'm glad we found this, and also wonder when `Plotly.newPlot` would ever be preferred over `Plotly.react`.

Reply:
https://plotly.com/javascript/plotlyjs-function-reference/#plotlyreact

If I read this correctly, I agree with you - `react` seems as though it would be preferable in all circumstances.  There must be some subtlety that is not reflected in the documentation where `newPlot` would be preferable.